addons/account/views/partner_view.xml

```diff
@@ -224,6 +224,7 @@
                                 <field name="currency_id" invisible="1"/>
                                 <field name="property_account_receivable_id"/>
                                 <field name="property_account_payable_id"/>
+                                <p colspan="2" class="text-info" attrs="{'invisible': ['!', '|', ('commercial_field_changed', 'ilike', 'property_account_receivable_id'), ('commercial_field_changed', 'ilike', 'property_account_payable_id')]}"><i class="fa fa-info-circle"/> Accounting-related changes will also be applied on <a role="button" name="open_commercial_entity" type="object" string="the parent company" class="oe_link"/>.</p>
```

Comment:
The warning will be shown even if you haven't modified the field, won't it?  The user has no visual feedback on _which_ fields are actually going to be written to the parent record.  Or is the message based on the assumption that those fields are supposed to have the same value on the parent record, and so being different implies upward modification?

Reply:
>  Or is the message based on the assumption that those fields are supposed to have the same value on the parent record, and so being different implies upward modification?

Indeed, since commercial fields are propagated at write on the parent to its children (that was the case since... time immemorial as far as i'm concerned) in the `_fields_sync` method, they are meant to always have similar values (the point of this PR is to establish upward propagation in addition to the downward one)

Reply:
in short: they cannot have different values (the same way related fields would not)